samples/02_commands_framework/Modules/PublicModule.cs

```diff
@@ -49,6 +50,29 @@ public async Task BanUserAsync(IGuildUser user, [Remainder] string reason = null
             await ReplyAsync("ok!");
         }
 
+        // Kick a user
+        [Command("kick")]
+        [RequireContext(ContextType.Guild)]
+        // make sure the user invoking the command can kick
+        [RequireUserPermission(GuildPermission.KickMembers)]
+        // make sure the bot itself can kick
+        [RequireBotPermission(GuildPermission.KickMembers)]
+        public async Task KickUserAsync(IGuildUser user, [Remainder] string reason = null)
+        {
+            // Creates the pattern TextBlock
+            EmbedBuilder builder = new EmbedBuilder();
+
+            builder.WithTitle(user.Username + " kicked from server!")
+                .WithColor(Color.Orange)
+                .WithDescription("by " +
+                                $"{Context.User.Username}" +
+                                Environment.NewLine +
+                                "Reason: " + reason);
+
+            await user.KickAsync(reason, null);
```

Comment:
`RequestOptions` is already `null` by default. If anything, show off how a `RequestOptions` can be used.

Reply:
RequestOptions is only really useful for setting audit log reasons, which is provided by the reason param anyway, IIRC.